### DISCUSSION TITLE
How actually does "only-new-issues" work?

### DISCUSSION THREAD

**Discussion by Sortren** [golangci/golangci-lint-action#981] [Q&A] [answered]
Due to lack of documentation regarding `only-new-issues` feature, I'd like to know how does it really work. 
I thought the desired workflow is following, considering such branch tree (topic has 3 commits A, B, C and open Pull Request on Github to desired master branch):

	  A---B---C topic
	 /
    D-- master

Whole pull request will be checked against a head of the master. So only those issues that come with the topic branch will be reported by the linter. 
In my case, the linter doesn't report anything even that I have issues done on purpose in the topic branch. 

To conclude, how does the `only-new-issues` work? I'd really like to go with the suggestion from the docs that says to not scan the whole repo but only the newest changes.

**Comment by ldez** [comment, marked as answer]
Hello,

I added a detailed explanation of the option: https://github.com/golangci/golangci-lint-action?tab=readme-ov-file#only-new-issues
